plugins/virus_scan/lib/batch/kVirusScanFlowManager.php

```diff
@@ -100,7 +100,15 @@ public function shouldConsumeAddedEvent(BaseObject $object)
 		if (!class_exists('kCurrentContext') || !kCurrentContext::isApiV3Context())
 			return false;
 		if($object instanceof asset)
+		{
+			$entryId = $object->getEntryId();
```

Comment:
Asset has a getentry method better use it.

Reply:
As Yossi suggested , just use $object->getentry()